### PR TITLE
Dev > Master for 1.13 release

### PR DESCRIPTION
**Pull request by drpatelh**
## PR checklist

 - [ ] This comment contains a description of changes (with reason)
 - [ ] `CHANGELOG.md` is updated
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
 # nf-core/tools: Changelog
 
+## v1.13dev
```
Comment (drpatelh) [inline CHANGELOG.md:3]:
```suggestion
## [v1.13 - Copper Crocodile](https://github.com/nf-core/tools/releases/tag/1.13) - [2021-03-16]
```

File: nf_core/modules/lint.py [hunk 2]
```diff
@@ -0,0 +1,801 @@
...
+                    url = module_base_url + f
+                    r = requests.get(url=url)
+
+                    if r.status_code != 200:
+                        self.warned.append(
+                            (
+                                mod,
+                                (
+                                    "check_local_copy",
+                                    f"Could not fetch remote copy, skippping comparison.",
```
Comment (phue) [inline nf_core/modules/lint.py:388]:
```suggestion
                                    f"Could not fetch remote copy, skipping comparison.",
```

**Review by KevinMenden — APPROVED** [review]

**Review by ewels — APPROVED** [review]
let's do this

**Comment by JoseEspinosa** [comment]
🎆

**Pull request merged**